packages/core/src/2d/assembler/TiledSpriteAssembler.ts

```diff
@@ -33,8 +34,15 @@ export class TiledSpriteAssembler {
     }
   }
 
-  static updatePositions(renderer: SpriteRenderer): void {
-    const { width, height, sprite, tileMode, tiledAdaptiveThreshold: threshold } = renderer;
+  static updatePositions(
```

Comment:
Same observation as mentioned earlier regarding the updatePositions function. Since the added parameters (width, height, pivot, flipX, and flipY) extend its functionality beyond just position updates, I would recommend renaming the function here as well to something like updateTransform or updateDimensionsAndPosition. This will help clarify its current role.